### DISCUSSION TITLE
incorrect cursor display

### DISCUSSION THREAD

**Discussion by Yorik1408** [DistroAV/DistroAV#1157] [Q&A - Support]
https://github.com/user-attachments/assets/08a1ee85-875a-480f-9065-dad3e8ab6eb9

![win](https://github.com/user-attachments/assets/78465d35-6f4a-4c31-89f3-8156b69e4a3b)
![mac](https://github.com/user-attachments/assets/8a1eb752-662c-44ef-acf8-47bda88daee8)

I'm not sure if the problem is in the plugin or OBS or NDI, but when I enable capture in OBS of my Macbook screen on my PC, the cursor is displayed in OBS incorrectly, not where it is on the Mac, but diagonally higher
Attaching a video of how they move Perhaps you will say that the problem is in the resolution, but then why is the cursor displayed correctly on the 11-inch MacBook, as well as on the Honor 15-inch laptop

**Comment by Trouffman** [comment]
This is not an issue by the receiving side ( DistroAV ) but by the software used to encode. :) if you used NDI Scan Converter on Mac this might be culprit.
